tests/performance/MemoryTest.cpp

```diff
@@ -393,18 +409,20 @@ TestConfiguration ShortRunningTestWithMemoryCache() {
  */
 TestConfiguration ShortRunningTestWithDiskCache() {
   using namespace olp;
-  auto cache_factory = []() {
-    cache::CacheSettings settings;
-    auto location = CustomParameters::getArgument("cache_location");
-    if (location.empty()) {
-      location = utils::Dir::TempDirectory() + "/performance_test";
-    }
-    settings.disk_path = location;
+
+  cache::CacheSettings settings;
+  auto location = CustomParameters::getArgument("cache_location");
+  if (location.empty()) {
+    location = utils::Dir::TempDirectory() + "/performance_test";
+  }
```

Comment:
Can you create the dir in case it does not exist?

Reply:
Our cache will do that